pkgs/development/python-modules/tpm2-pytss/default.nix

```diff
@@ -27,9 +28,8 @@ buildPythonPackage rec {
 
   nativeBuildInputs = [
     cffi
-    pkgconfig
-    # somehow propagating from pkgconfig does not work
-    pkg-config
+    pkgconfig # this is the python module
+    pkg-config # this is the actual pkg-config tool
```

Comment:
pkgconfig should actually contain pkg-config already or are they shelling out? Also we don't really need those comments but whatever.

Reply:
> pkgconfig should actually contain pkg-config already or are they shelling out? 

No idea what they are doing, but having both makes it work (this was also the case previously).

> Also we don't really need those comments but whatever.

Oh, we absolutely do. If anything we need a lot *more* comments in nixpkgs in general. Whenever anything unusual is done, we should definitely add  a comment.

Reply:
> No idea what they are doing, but having both makes it work (this was also the case previously).

pkgconfig should propagate pkg-config into nativeBuildInputs. That only works sometimes and I gave up debugging why it not always works.